docs/changelog.rst

```diff
@@ -1,5 +1,15 @@
 :orphan:
 
+2.9.2
+=======
+- TwitchIO
+    - Changes:
+        - `~twitchio.PartialUser.fetch_moderated_channels` returns "broadcaster_login" instead of "broadcaster_name"
```

Comment:
```suggestion
        - :func:`twitchio.PartialUser.fetch_moderated_channels <PartialUser.fetch_moderated_channels>` returns "broadcaster_login" api field instead of "broadcaster_name"
```